### PR TITLE
feat: support local fal-scripts-path on remote isolate

### PR DESCRIPTION
**Pull request by isidentical**
### Integration tests

Adapter to test:

- postgres

Python version to test:

- 3.8

### CODE REVIEW

File: adapter/src/dbt/adapters/fal_experimental/adapter.py [hunk 1]
```diff
@@ -33,51 +39,90 @@ def run_with_adapter(code: str, adapter: BaseAdapter, config: RuntimeConfig) ->
...
     # This function can be run in an entirely separate
     # process or an environment, so we need to reconstruct
     # the DB adapter solely from the config.
     adapter = reconstruct_adapter(config, manifest, macro_manifest)
+    fal_scripts_path = get_fal_scripts_path(config)
+    if local_packages is not None:
+        assert not fal_scripts_path.exists()
+        fal_scripts_path.parent.mkdir(parents=True, exist_ok=True)
+        zip_file = zipfile.ZipFile(io.BytesIO(local_packages))
+        zip_file.extractall(fal_scripts_path)
```
Comment (chamini2) [inline adapter/src/dbt/adapters/fal_experimental/adapter.py:59]: pretty cool!

**Review by chamini2 — APPROVED** [review]
can we add a test?

**Review by turbo1912 — APPROVED** [review]

**Review by mederka — APPROVED** [review]
👏

**Comment by mederka** [comment]
Do we know what's up with bq tests?

**Comment by isidentical** [comment]
Now included under Meder's PR #683

**Pull request closed without merging**